docs/otel/sdk-metrics.md

```diff
@@ -882,5 +887,136 @@ E.g. for Java the fully qualified classname SHOULD be used in this case.
 <!-- END AUTOGENERATED TEXT -->
 <!-- endsemconv -->
 
+## Operation Metrics
+
+### Metric: `otel.sdk.exporter.operation.duration`
+
+This metric is [opt-in][MetricOptIn].
```

Comment:
Discussed at the SemConv SIG meeting 4/7/25:

- this is an important metrics that allows to count number of export calls (not a number of spans) and final (after all tries) success rate. It should be recommended
- to offset the costs associated with histogram, we should consider reducing the number of buckets. E.g. we've done 4 buckets in [JVM GC duration](https://github.com/open-telemetry/semantic-conventions/blob/c1d8d371e8ad7cf47b50b6e0dd976746acdaeddf/docs/runtime/jvm-metrics.md#metric-jvmgcduration). We can even start with one bucket here. 

/cc @JonasKunz to keep me honest here

PS: if we go ahead with one bucket, I'd suggest `1` sec as something I would use for P95 export duration target.

Reply:
> PS: if we go ahead with one bucket, I'd suggest `1` sec as something I would use for P95 export duration target.

just a terminology clarification, on the SIG call I was thinking one bucket histogram would mean no boundaries

Reply:
@trask 

Just to clarify, does this me a `ExplicitBucketBoundaries` defined with `[double.max]` as a single entry?

Reply:
I believe it's an empty array which results in a single bucket covering -Inf, Inf range. You still get count and sum, i.e. average from it.

Reply:
@lzchen could you please also make the metric recommended? Thanks!